### ISSUE TITLE
DE: Plasma and WM: i3 consult

### ISSUE THREAD

**Issue by Ynjxsjmh** [ClydeDroid/dotfiles#2]
Currently my DE is Plasma and WM is KWin, I want to change my WM to i3. Some threads [1,2,3] use `wmctrl` to kill the Plasma desktop view. You seem use a different way since I didn't find `wmctrl` in your i3 config file. Could you share your experience on how to disable KWin and use i3 instead?

Best regards.

1. [/u/shinyeye4's config](https://www.reddit.com/r/unixporn/comments/64mihc/i3_kde_plasma_a_match_made_in_heaven/dg4k0wq?utm_source=share&utm_medium=web2x&context=3) | [i3 + Plasma](https://github.com/avivace/dotfiles#i3--plasma-integration)
2. [Replacing KWin by i3 / Applications &amp; Desktop Environments / Arch Linux Forums](https://bbs.archlinux.org/viewtopic.php?id=227301)
3. [Tutorials/Using Other Window Managers with Plasma - KDE UserBase Wiki](https://userbase.kde.org/Tutorials/Using_Other_Window_Managers_with_Plasma)

**Comment by nickclyde** [comment]
Hi @Ynjxsjmh! Sorry I took a while to get back to you. I have sinced moved on and got rid of Plasma - I'm only using i3 now. (I should probably update this repo, lol)

But from what I recall when I was doing this, I added `exec i3` to my `.xinitrc` file to use i3 instead of KWin. This might be helpful too:
https://wiki.archlinux.org/index.php/Xinit#Switching_between_desktop_environments/window_managers
